beater/sipcmbeat.go

```diff
@@ -742,16 +710,15 @@ func (bt *Sipcmbeat) getCallID(dst, src []byte, callID sipsp.PField, encFlags *F
 	if bt.Config.UseCallIDAnonymization() && (len(src) > 0) {
 		// anonymize Call-ID
 		//anonymization.DbgOn()
-		ac := anonymization.AnonymPField{
-			PField: callID,
-		}
-		if err := ac.Anonymize(dst, src); err != nil {
+		bt.anonymizer.CallId.PField = callID
+		aCallId, err := bt.anonymizer.CallId.Anonymize(dst, src)
+		if err != nil {
```

Comment:
not really safe since `CallId.Anonymize()` changes internally the `PField` to `{0, len(src)}`.